### DISCUSSION TITLE
ssh

### DISCUSSION THREAD

**Discussion by chaimt** [homeassistant-apps/app-cloudflared#676] [Ideas]
does your applicaiton support ssh tuneeling

**Comment by elcajon** [comment]
Yes, you can configure ssh either via additional_hosts (with a `service` like `ssh://192.168.0.1:22`) or you can use a remotely managed tunnel.

See also https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/use-cases/ssh/

**Reply by tomasbedrich** [reply]
Hello, I am struggling with the setup... Can you please elaborate on what needs to be set up in the Cloudflare dashboard for this to work?
